lib/v2/datasource/remote/model/token_model.dart

```diff
@@ -1,30 +1,60 @@
-class TokenModel {
+import 'package:equatable/equatable.dart';
+
+class TokenModel extends Equatable {
   final String chainName;
   final String contract;
   final String symbol;
   final String name;
+  final String backgroundImage;
+  final String logo;
+  final String balanceSubTitle;
+
   const TokenModel({
     required this.chainName,
     required this.contract,
     required this.symbol,
     required this.name,
-    });
+    required this.backgroundImage,
+    required this.logo,
+    required this.balanceSubTitle,
+  });
+
+  @override
+  List<Object?> get props => [chainName, contract, symbol];
 }
 
 const SeedsToken = TokenModel(
-  chainName: "Telos", 
-  contract: "token.seeds", 
-  symbol: "SEEDS", 
-  name: "Seeds");
+    chainName: "Telos",
+    contract: "token.seeds",
+    symbol: "SEEDS",
+    name: "Seeds",
+    backgroundImage: 'assets/images/wallet/currency_info_cards/seeds/background.jpg',
```

Comment:
these list of tokens would be nice to have them stored in chain or firebase
Not for this PR tho.

Reply:
For sure. Or on chain.